pages/index.js

```diff
@@ -6,15 +6,26 @@ const StyledHeader = styled.h1`
   font-size: 2.5rem;
 `;
 
-export default function HomePage({ onMarkCorrect, flashCards, collections }) {
+export default function HomePage({
+  onMarkCorrect,
+  deleteCard,
+  flashCards,
+  collections,
+  noCards,
+}) {
   return (
     <main>
       <StyledHeader>FlipWise</StyledHeader>
```

Comment:
The name suggests that this is a `header` element but instead it is an `h1`. Therefore I suggest to rename the styled component to `StyledHeading`.